catalog/index.yaml

```diff
@@ -0,0 +1,243 @@
+        and so on.
+
+      # How to install
+        Read the information about the Operator and click Install.
+
+        On the Install Operator page:
+
+        - Select `candidate` from the list of available Update Channel options.
+        This ensures that you install the latest version of OpenShift sandboxed containers
+        that is compatible with your OpenShift Container Platform version.
+
+        - For Installed Namespace, ensure that the Operator recommended namespace
+          option is selected. This installs the Operator in the mandatory
+          `openshift-sandboxed-containers-operator` namespace, which is automatically
+          created if it does not exist. Attempting to install the OpenShift
+          sandboxed containers Operator in a namespace other than
+          `openshift-sandboxed-containers-operator` causes the installation to fail.
+
+        - For Approval Strategy, ensure that Automatic, which is the default value,
+          is selected. OpenShift sandboxed containers automatically updates when a new
+          z-stream release is available.
+
+        - Click Install to make the Operator available to the OpenShift sandboxed
+          containers namespace.
+
+        - The OpenShift sandboxed containers Operator is now installed on your
+          cluster. You can trigger the Operator by enabling the runtime on your cluster.
+          You can do this by creating a `KataConfig` CustomResourceDefinition(CRD) instance. For this click
+          on "create instance" on the operator overview page.
+
+      # Documentation
+      See the official documentation [here](https://docs.openshift.com/container-platform/latest/sandboxed_containers/understanding-sandboxed-containers.html)
+    displayName: OpenShift sandboxed containers Operator
+    installModes:
+    - supported: true
+      type: OwnNamespace
+    - supported: true
+      type: SingleNamespace
+    - supported: false
+      type: MultiNamespace
+    - supported: false
+      type: AllNamespaces
+    keywords:
+    - sandboxed-containers
+    - kata
+    labels:
+      operatorframework.io/arch.amd64: supported
+      operatorframework.io/os.linux: supported
+    links:
+    - name: Sandboxed Containers Operator
+      url: https://www.github.com/openshift/sandboxed-containers-operator
+    maintainers:
+    - email: support@redhat.com'
+      name: '''Red Hat'
+    maturity: beta
+    provider:
+      name: Red Hat
+relatedImages:
+- image: gcr.io/kubebuilder/kube-rbac-proxy@sha256:d99a8d144816b951a67648c12c0b988936ccd25cf3754f3cd85ab8c01592248f
+  name: ""
+- image: quay.io/openshift_sandboxed_containers/openshift-sandboxed-containers-operator-bundle:v1.6.0
+  name: ""
+- image: quay.io/openshift_sandboxed_containers/openshift-sandboxed-containers-operator:v1.6.0
+  name: ""
+- image: registry.redhat.io/openshift-sandboxed-containers/osc-cloud-api-adaptor-rhel9:latest
```

Comment:
We try to avoid using the `latest` tag as explained in f8169dbedc131b60c1268c7757a0afd84523498b.

Reply:
The `latest` versioning for the podvm probably comes from` config/peerpods/podvm`.
We will have to investigate the others